editor.planx.uk/src/@planx/components/PlanningConstraints/Editor.tsx

```diff
@@ -1,15 +1,25 @@
+import Box from "@mui/material/Box";
+import Checkbox from "@mui/material/Checkbox";
```

Comment:
design: We currently use a number of different components for controlling boolean values in the Editor - 
 - MUI switches (in design settings)
 - PlanX checkboxes (in `Send` component)
 - MUI checkboxes here

This is a fix forward / design question which @ianjon3s may want his eyes on rather than a recommendation to make this change in the current PR 👍